ppstructure/recovery/recovery_to_markdown.py

```diff
@@ -157,7 +157,12 @@ def replace_special_char(content):
                 f"""<div align="center">\n\t<img src="{img_name+"/"+img_file_name}">\n</div>"""
             )
         elif region["type"].lower() == "title":
-            markdown_string.append(f"""# {region["res"][0]["text"]}""")
+            markdown_string.append(
+                f"""# {region['res'][0]['text']}"""
+                + "".join(
+                    [" " + one_region["text"] for one_region in region["res"][1:]]
```

Comment:
Is it possible for `region["res"]` to contain only one element, and could this lead to potential out-of-bounds issues?

Reply:
That's no problem. Most of results are cases as you said. like:
```python
>>> test = [{"text": "abc"}]
>>> region = {"res": test}
>>> markdown_string = []
>>> markdown_string.append(
...    f"""# {region['res'][0]['text']}"""
...     +  "".join(
...                     [" " + one_region["text"] for one_region in region["res"][1:]])
... )
>>> markdown_string
['# abc']
```